### PR TITLE
update calendar date text color

### PR DESCRIPTION
**Pull request by chiuam**
### Platforms Impacted
- [X] iOS
- [ ] macOS

### Description of changes
Update the calendar past date text color, it should be Colors.textPrimary for light mode and Colors.textSecondary dark mode.

### Verification
Tested dark and light mode on the test app.

| Before  
image
image

| After                                      
image
image

### Pull request checklist

This PR has considered:
- [X] Light and Dark appearances
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/159)

### CODE REVIEW

File: ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import UIKit
 public extension Colors {
     struct Calendar {
         public struct Day {
-            public static var textPrimary = UIColor(light: Colors.textSecondary, dark: Colors.textPrimary)
+            public static var textPrimary: UIColor = Colors.textPrimary
             public static var textSecondary: UIColor = Colors.textSecondary
```
Comment (harrieshin) [inline ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift:14]: no this is one that should change

Comment (chiuam) [reply ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift:14]: Sorry, I don't follow. Which one do you mean that should change?

File: Changelist21127429.log [hunk 2]
```diff
@@ -0,0 +1,18 @@
+resolve3 /Users/amchiu/apx/src/temp/pak90626.0/00_FREConsentCoordinator.m_v17 /Users/amchiu/apx/src/temp/pak90626.0/00_FREConsentCoordinator.m_v17_changed /Users/amchiu/apx/src/dev/platformsdk/apple/ui/setupui/subscriptionui_ios/src/FREConsentCoordinator.m /Users/amchiu/apx/src/temp/pak90626.0/00_FREConsentCoordinator.m_v17_merged (copy theirs)
```
Comment (harrieshin) [inline Changelist21127429.log:1]: this shouldn't be in the PR

File: ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift (hunk 1 above)
Comment (chiuam) [reply ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift]: Resolved offline. I should be updating the past date color, not the future date color.

File: Changelist21127429.log (hunk 2 above)
Comment (Saadnajmi) [reply Changelist21127429.log:1]: You can blame me for that, I taught Amanda `git add *` 😅

Comment (harrieshin) [reply Changelist21127429.log]: always check ur changes before creating a PR  :P

**Pull request merged**